salt/tests/requirements.in

```diff
@@ -6,3 +6,4 @@ parameterized == 0.7.4
 etcd3 != 0.11.0
 kubernetes
 urllib3
+pyfakefs
```

Comment:
Cool stuff. Could be useful to reuse it in other tests, one day :innocent:

Reply:
Yeah, I'm so glad I remembered this stuff (from a conversation we had few months ago), at the beginning I was trying to mock everything pathlib related, it was ugly then I used this and... there is almost nothing to do and everything work like a charm.